nodes/srl/srl.go

```diff
@@ -124,11 +124,12 @@ func (s *srl) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 	s.DefaultNode = *nodes.NewDefaultNode(s)
 	// set virtualization requirement
 	s.HostRequirements.SSSE3 = true
+	s.HostRequirements.MinVCPU = 2
+	s.HostRequirements.MinVCPUFailAction = types.FailBehaviourLog
+	s.HostRequirements.MinFreeMemoryGb = 2
+	s.HostRequirements.MinFreeMemoryGbFailAction = types.FailBehaviourLog
```

Comment:
@steiler I think in srl case we should error and stop execution if vCPU < 2
we should keep logging action for memory, as a single node might be able to start with 2G